client/.idea/encodings.xml

```diff
@@ -1,6 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think they may have been added upon the creation of the french values directory.